.changeset/clean-geese-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@withease/web-api': patch
+---
+
+Add `$isLandscapeOrientation` and `$isPortraitOrientation` stores to `trackScreenOrientation` integration
```

Comment:
🤓 for me, the next spell `trackScreenOrientation(...).$isLandscapeOrientation` is a little bit excessive. Could we name it like `$landscape`?

Reply:
[Renamed](https://github.com/igorkamyshev/withease/pull/70/commits/02c71287e8e965944e64a68e1ec75295c44116e6) new stores to suggested format